src/Namshi/JOSE/SimpleJWS.php

```diff
@@ -29,12 +29,14 @@ public function __construct($header = array(), $encryptionEngine = 'OpenSSL')
      * Sets the payload of the current JWS with an issued at value in the 'iat' property.
      *
      * @param array $payload
+     *
+     * @return $this
      */
     public function setPayload(array $payload)
     {
         if (!isset($payload['iat'])) {
             $now = new \DateTime('now');
-            $payload['iat'] = $now->format('U');
+            $payload['iat'] = $now->getTimeStamp();
```

Comment:
`$payload['iat'] = time();` gives the same result as `$now = new \DateTime('now'); $payload['iat'] = $now->getTimeStamp();` so why use two lines with a `DateTime` object instantiation instead of a simple PHP function?